common/helpers/helperMethods.ts

```diff
@@ -0,0 +1,15 @@
+import {User} from "../models/user";
+
+export class helperMethods{
+    public static getMinuteUnit(estimatedDuration: number) {
```

Comment:
I don't know that this helpermethods file is really needed. We can move both of these functions to the respective classes.

Reply:
3am Anna thought this was a good idea. Logic was not in the equation at the time of this decision. Will fix.